src/preact.d.ts

```diff
@@ -455,6 +455,7 @@ declare global {
 			onSearchCapture?: GenericEventHandler;
 			onSubmit?: GenericEventHandler;
 			onSubmitCapture?: GenericEventHandler;
+	                onInvalid?: GenericEventHandler;
```

Comment:
Seems like the indentation is off. I'm happy to merge it once that's fixed 👍

Reply:
@marvinhagemeister Done. Let me know if any other changes are required.

Reply:
Awesome, thanks for making these changes 👍